# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-02-22 17:48+0100\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=CHARSET\n"
"Content-Transfer-Encoding: 8bit\n"

#: ../usr/lib/ddm/ddm.py:46 ../usr/lib/ddm/main.py:26
#: ../usr/share/applications/ddm.desktop.in.h:1
msgid "Device Driver Manager"
msgstr ""

#: ../usr/lib/ddm/ddm.py:47 ../usr/lib/ddm/ddm.py:155
msgid "Install"
msgstr ""

#: ../usr/lib/ddm/ddm.py:48
msgid "Help"
msgstr ""

#: ../usr/lib/ddm/ddm.py:49
msgid "Quit"
msgstr ""

#: ../usr/lib/ddm/ddm.py:104
msgid "No internet connection"
msgstr ""

#: ../usr/lib/ddm/ddm.py:105
msgid ""
"You need an internet connection to install the additional software.\n"
"Please, connect to the internet and try again."
msgstr ""

#: ../usr/lib/ddm/ddm.py:155
msgid "Device"
msgstr ""

#: ../usr/lib/ddm/ddm.py:171
msgid "Remove kernel"
msgstr ""

#: ../usr/lib/ddm/ddm.py:172
msgid ""
"You cannot remove a booted kernel.\n"
"Please, boot another kernel and try again."
msgstr ""

#: ../usr/lib/ddm/ddm.py:193
msgid "Hardware scan"
msgstr ""

#: ../usr/lib/ddm/ddm.py:197
msgid "There are no available drivers for your hardware:"
msgstr ""

#: ../usr/lib/ddm/ddm.py:203
msgid "DDM did not find any supported hardware."
msgstr ""

#: ../usr/lib/ddm/ddm.py:251
msgid "Saved"
msgstr ""

#: ../usr/lib/ddm/ddm.py:252
msgid "You will need to restart your system."
msgstr ""

#: ../usr/lib/ddm/ddm.py:456
msgid "PAE capable system"
msgstr ""

#: ../usr/lib/ddm/ddm.py:558
msgid ""
"There was an error during the installation.\n"
"Please, run 'sudo apt-get -f install' in a terminal.\n"
"Visit our forum for support: http://forums.solydxk.com"
msgstr ""

#: ../usr/lib/ddm/ddm.py:567
msgid "The software has been successfully installed."
msgstr ""

#: ../usr/lib/ddm/main.py:27
msgid ""
"Device Driver Manager cannot be started in a live environment\n"
"You can use the --force argument to start DDM in a live environment"
msgstr ""

#: ../usr/lib/ddm/main.py:69
msgid "Unexpected error"
msgstr ""

#: ../usr/lib/ddm/main.py:70
msgid ""
"<b>DDM has failed with the following unexpected error. Please submit a bug "
"report!</b>"
msgstr ""
